app/.#post.go

```diff
@@ -0,0 +1 @@
+tomas@tomas.6171:1559840605
```

Comment:
This file should not be here :wink:

Reply:
sorry that my IDE playing tricks. removed.